config.md

```diff
@@ -69,6 +69,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 
 * **`destination`** (string, REQUIRED) Destination of mount point: path inside container.
     This value MUST be an absolute path.
+    * Linux: It MAY be a relative path too, in which case it MUST be relative to "/".
```

Comment:
Just because I know the problems with the ecosystem exist on linux and I don't know how to do it on Windows (shall we say relative to C:? Or to D:?)

Reply:
for windows (haven't looked yet at our implementations), but at least I know that Windows containers by default only have a single drive (so only `C:\`) and any path should be on the `C:\` drive)

Reply:
I've updated. I kept it limited to Linux as that is the only platform I know it can cause issues. If others find out later that other platforms are also affected, we can relax them later too :)

Reply:
I've also mentioned relative paths are deprecated. Let me know what you think!